test/config/fixtures.ts

```diff
@@ -0,0 +1,27 @@
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import * as sinonChai from 'sinon-chai';
+import * as chaiThings from 'chai-things';
+import * as chaiLike from 'chai-like';
+
+/**
+ * Mocha runs this EXACTLY ONCE before all tests to set up globals that
+ * are used among all test files, such as chai assertion plugins. See
+ * https://mochajs.org/#test-fixture-decision-tree-wizard-thing
+ *
+ * IMPORTANT: The functionality in this global fixture hook should not break any
+ * suite run in isolation, and it should be required by ALL test suites.
+ * If unsure whether to add to global fixtures, refer to the chart above.
+ * Also, avoid setting global mutable variables here.
+ */
+export const mochaGlobalSetup = function () {
+	console.log('Setting up global fixtures for tests...');
+
+	/* Setup chai assertion plugins */
+	chai.use(chaiAsPromised);
+	chai.use(sinonChai);
+	// tslint:disable-next-line
+	chai.use(require('chai-events'));
```

Comment:
Can't you do `import * as chaiEvents` above and then do `chai.use(chaiEvents)` here?

Did you try removing that dependency? I see a couple of test suites that import it but I don't see any of the assertions related to the package https://www.npmjs.com/package/chai-events

Reply:
Sounds good @pipex, looks like it's being imported in a number of places but the assertions are not used.

Reply:
I've scrubbed all mentions of chai-events from the codebase :)